rotkehlchen/tests/api/test_history.py

```diff
@@ -95,7 +96,7 @@ def test_query_history(rotkehlchen_api_server_with_exchanges, start_ts, end_ts):
     assert settings['eth_staking_taxable_after_withdrawal_enabled'] is False
 
     assert events_result['entries_limit'] == FREE_PNL_EVENTS_LIMIT
-    entries_length = 43 if start_ts == 0 else 40
+    entries_length = 35 if start_ts == 0 else 32
```

Comment:
Is this reduction due to the fees being included in the cost basis and thus not having standalone events? Couldn't this be made configurable here to also see standalone event approach still works?

Reply:
Yes, is because now there are no fee events.

I'll check, but it should be easy to parametrize this test.